typings/balena-sdk/models.d.ts

```diff
@@ -352,11 +352,21 @@ export interface Device {
 	gateway_download?: ReverseNavigationResource<GatewayDownload>;
 }
 
+export interface CpuArchitecture {
```

Comment:
Please also add this to the `ResourceTypeMap` so that the sdk can infer the type from the resource name.